core/src/martian/swagger.cljc

```diff
@@ -45,33 +45,35 @@
             %) parts)))
 
 (defn- ->handler [{:keys [definitions] :as swagger-map} url-pattern [method swagger-definition]]
-  (let [path-parts (tokenise-path url-pattern)
-        uri (string/join (map str path-parts))
-        parameters (:parameters swagger-definition)]
-    {:path uri
-     :path-parts path-parts
-     :method method
-     :path-schema (path-schema definitions parameters)
-     :query-schema (query-schema definitions parameters)
-     :body-schema (body-schema definitions parameters)
-     :form-schema (form-schema definitions parameters)
-     :headers-schema (headers-schema definitions parameters)
-     :response-schemas (response-schemas definitions (:responses swagger-definition))
-     :produces (some :produces [swagger-definition swagger-map])
-     :consumes (some :consumes [swagger-definition swagger-map])
-     :swagger-definition swagger-definition
-     ;; todo path constraints - required?
-     ;; :path-constraints {:id "(\\d+)"},
-     ;; {:in "path", :name "id", :description "", :required true, :type "string", :format "uuid"
-     :route-name (->kebab-case-keyword (:operationId swagger-definition))}))
```

Comment:
my use case: (->kebab-case-keyword nil) throws, when first processing the swagger spec.

could replace with (some-> (:operationId swagger-definition) ->kebab-case-keyword), but I'm not sure how useful having an entry with no route-name is, hence my initial thought to filter out when no operationId.